views/dist/front.js

```diff
@@ -1,3 +1,3 @@
al(),n[","]=e.getGroup(),n["-"]=e.getMinusSign(),n["%"]=e.getPercentSign(),n["+"]=e.getPlusSign(),this.strtr(t,n)}},{key:"strtr",value:function(t,e){var n=Object.keys(e).map(k);return t.split(RegExp("(".concat(n.join("|"),")"))).map((function(t){return e[t]||t})).join("")}},{key:"addPlaceholders",value:function(t,e){return e.replace(/#?(,#+)*0(\.[0#]+)*/,t)}},{key:"performSpecificReplacements",value:function(t){return this.numberSpecification instanceof b?t.split("¤").join(this.numberSpecification.getCurrencySymbol()):t}}],r=[{key:"build",value:function(e){var n;return n=void 0!==e.numberSymbols?S(l,P(e.numberSymbols)):S(l,P(e.symbol)),new t(e.currencySymbol?new b(e.positivePattern,e.negativePattern,n,parseInt(e.maxFractionDigits,10),parseInt(e.minFractionDigits,10),e.groupingUsed,e.primaryGroupSize,e.secondaryGroupSize,e.currencySymbol,e.currencyCode):new p(e.positivePattern,e.negativePattern,n,parseInt(e.maxFractionDigits,10),parseInt(e.minFractionDigits,10),e.groupingUsed,e.primaryGroupSize,e.secondaryGroupSize))}}],n&&E(e.prototype,n),r&&E(e,r),Object.defineProperty(e,"prototype",{writable:!1}),t}();const I=M;var _={},C=function(t,e,n,r){void 0===_[t]?e.text(e.text().replace(/([^\d]*)(?:[\d\s.,]+)([^\d]+)(?:[\d\s.,]+)(.*)/,"$1".concat(n,"$2").concat(r,"$3"))):e.text("".concat(_[t].format(n)," - ").concat(_[t].format(r)))};const D=function(){$(".faceted-slider").each((function(){var t=$(this),n=t.data("slider-values"),r=t.data("slider-specifications");null!=r&&(_[t.data("slider-id")]=I.build(r)),C(t.data("slider-id"),$("#facet_label_".concat(t.data("slider-id"))),null===n?t.data("slider-min"):n[0],null===n?t.data("slider-max"):n[1]),$("#slider-range_".concat(t.data("slider-id"))).slider({range:!0,min:t.data("slider-min"),max:t.data("slider-max"),values:[null===n?t.data("slider-min"):n[0],null===n?t.data("slider-max"):n[1]],stop:function(n,r){var i=t.data("slider-encoded-url").split("?"),o=[];i.length>1&&(o=e(i[1]));var a=!1;o.forEach((function(t){"q"===t.name&&(a=!0)})),a||o.push({name:"q",value:""}),o.forEach((function(e){"q"===e.name&&(e.value+=[e.value.length>0?"/":"",t.data("slider-label"),"-",t.data("slider-unit"),"-",r.values[0],"-",r.values[1]].join(""))}));var u=[i[0],"?",$.param(o)].join("");prestashop.emit("updateFacets",u)},slide:function(e,n){C(t.data("slider-id"),$("#facet_label_".concat(t.data("slider-id"))),n.values[0],n.values[1])}})}))};var F=n(379),T=n.n(F),G=n(580),A=n.n(G);T()(A(),{insert:"head",singleton:!1}),A().locals,$(document).ready((function(){prestashop.on("updateProductList",(function(){$(".faceted-overlay").remove(),D()})),D(),prestashop.on("updateFacets",(function(){1!==$(".faceted-overlay").length&&$("body").append('<div class="faceted-overlay">\n<div class="overlay__inner">\n<div class="overlay__content"><span class="spinner"></span></div>\n</div>\n</div>')}))}));var R=n(765),z=n.n(R);T()(z(),{insert:"head",singleton:!1}),z().locals;var U=n(741),N=n.n(U);T()(N(),{insert:"head",singleton:!1}),N().locals})()})();
```

Comment:
Do we need to version this file?

Reply:
It is unneeded if `npm run build` is run automatically in workflows.